data/input/court.mainnet.js

```diff
@@ -11,13 +11,13 @@ const START_DATE = Math.floor(new Date('2020-02-10T16:00:00.000Z') / 1000)
 const ANJ = {
   symbol: 'ANJ',
   decimals: 18,
-  address: requireOutput('minime.mainnet', tokens => tokens.ANJ.address)
+  address: requireOutput('minime.mainnet', getAddressIfDefined('ANJ'))
```

Comment:
I guess it’s fine here too, but the I only added `getAddressIfDefined` in places where there was a loop over all networks because missing for instance the value for `rpc` was making deployment on Rinkeby fail. So I don’t think we need it here, but I guess it doesn’t hurt.